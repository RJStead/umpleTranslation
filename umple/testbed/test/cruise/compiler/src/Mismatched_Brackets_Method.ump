generate Java "../src-gen-umple";

class MismatchedBrackets
{
    public double foo() { 
        for (int i = 0; i < 10; i++) {
            if (i % 2)) {
                return i / 2.0;
            }
        }
        return 0.0;
    }
}
