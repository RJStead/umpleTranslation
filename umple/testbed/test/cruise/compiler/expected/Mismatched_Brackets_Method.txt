Mismatched_Brackets_Method.ump:7: error: illegal start of expression
            if (i % 2)) {
                      ^
1 error